app/config/config.php

```diff
@@ -83,6 +83,30 @@
     ],
     'fragments'            => null,
     'http_method_override' => true,
+    'messenger'            => [
+        'default_bus' => 'email.bus',
+        'buses'       => [
+            'email.bus' => null,
+        ],
+        'transports'  => [
+            'email_transport' => [
+                'dsn'            => '%env(MAUTIC_MESSENGER_TRANSPORT_DSN)%',
+                'options'        => [
+                    'table_name' => MAUTIC_TABLE_PREFIX.'messenger_messages',
```

Comment:
I am using this custom definition of the table name to include the prefix of the table according to these [docs](https://symfony.com/doc/4.4/messenger.html#doctrine-transport)